src/testdir/test_functions.vim

```diff
@@ -1237,7 +1237,8 @@ func Test_platform_name()
   if has('unix') && executable('uname')
     let uname = system('uname')
```

Comment:
If we used `uname -s` instead of `uname` (which I assume is portable), we would not need to use regexp matching, for some of the checks below.  We could write:
```
call assert_equal(uname ==# 'BeOS', has('beos'))
```

Reply:
`uname` without any flags is the same as `uname -s`.